crypto/x509v3/v3_ncons.c

```diff
@@ -9,6 +9,7 @@
 
 #include <stdio.h>
 #include "internal/cryptlib.h"
+#include "e_os.h"
```

Comment:
Does it make sense to create a new header file for all string,strings,memcmp kinds of things?  I'm thinking pull the malloc stuff out of crypto.h, the ctype, etc., stuff out of e_os.h and so on.  It would be a separate PR, but just a thought.